helios/media/static_templates/question.html

```diff
@@ -1,10 +1,20 @@
-
 {#foreach $T.questions as question}
 <div id="q_view_{$T.question$index}">
+<h4><strong>{$T.question$index + 1}&ordf; Question</strong>
```

Comment:
not sure this big question title is necessary, could we just remove it and have the question itself serve as the title?